jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = Object.assign({}, config, {
   moduleNameMapper: {
     ...internalModules,
     '\\.(css|less)$': 'empty/object',
-    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': 'empty/object'
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$': 'empty/object'
```

Comment:
Would have preferred an explicit entry so this doesn't get lost inside "media types".